### DISCUSSION TITLE
RU nicht erreichbar

### DISCUSSION THREAD

**Discussion by pbroska** [Digitaler-Impfnachweis/certification-apis#123] [Q&A] [answered]
Heute kam endlich der Test-Secunet Konnektor für die RU. VPN ist verbunden, Rechner läuft über den Konnektor (Standard Gateway und DNS). Trotzdem ist die RU nicht erreichbar. Internetzugang über den Konnektor ist vorhanden. Welchen route add Befehl muss ich verwenden? Alle bereits genannten habe ich probiert: route add 100.102.0.0 mask 255.255.0.0 192.168.10.122

curl --request GET "https://id.ru.impfnachweis.info/auth/realms/ubirch-certify/protocol/openid-connect/auth?redirect_uri=connector://authenticated&response_type=code&scope=openid&client_id=user-access-ti-kops&nonce=1" -v
Note: Unnecessary use of -X or --request, GET is already inferred.
* Could not resolve host: id.ru.impfnachweis.info
* Closing connection 0
curl: (6) Could not resolve host: id.ru.impfnachweis.info

**Comment by twwolff** [comment]
Hallo für die RU die route soll so sein : route add 10.30.17.10 mask 255.255.255.255 [Connector ip adress] , aber wenn es schon die Standard Gateway ist sollte theoretisch klappen. Ip from ru is 10.30.17.10 . (api.ru.impfnachweis.info). Aber die Test-Server funktionieren sehr schlecht. Ich habe den ganze Tag ein abfrage probiert, und habe ich nur einmal ein Antwort bekommen. (sogar Leider ein leere PDF ! )

**Reply by pbroska** [reply]
Danke für die Info, dann werde ich mal etwas warten. Kriege immer noch keinen connect und werde berichten ... VG Patrick

**Reply by pbroska** [reply]
Manuels Youtube Video brachte den Hinweis. In der hosts Datei hinzufügen:
10.30.17.10 id.ru.impfnachweis.info
und funktioniert! Danke nochmal an Manuel Blechschmidt!

**Comment by pbroska** [comment, marked as answer]
Siehe reply
